cfpb/cfgov-refresh
# Mega Menu: Fix JS error when resizing

The mega menu body click listener of the desktop menu was not being removed when suspending the desktop menu, which in some circumstances led to an error in the mobile menu.

## Changes

- When resizing, call the update menu state function with no menu to clean up its state.


## How to test this PR

To repeat error:
1. Visit https://www.consumerfinance.gov
2. Click to open the menu.
3. Inspect the page to view the dev console.
4. Resize to mobile size.
5. Click to open the hamburger menu.
6. See error `Uncaught TypeError: Cannot read property 'getTransition' of null`

Do the same thing in this PR and see that the error is gone.
